README.md

```diff
@@ -38,13 +38,14 @@ sudo apt update
 ./install.sh --update
 ```
 
-Use `./{game}/install-{game}.sh` to update only one game.
+Use `./{game}/install.sh` to update only one game.
 
 ### Game servers management
 
-```sh
-sudo systemctl {start, stop or restart} {game}-server.service
-```
+`sudo service {game}-server {start|stop|restart|status}`
+or
+`sudo systemctl {start|stop|restart|status} {game}-server.service`
+
```

Comment:
In this case, I prefer block code instead of inline code.

``````suggestion
```sh
sudo service {game}-server {start|stop|restart|status}
```

or

```sh
sudo systemctl {start|stop|restart|status} {game}-server.service
```
``````

Reply:
Fair enough.